examples/mlperf/optimizers.py

```diff
@@ -0,0 +1,37 @@
+from typing import List, Set
+
+from tinygrad import Tensor
+from tinygrad.nn.optim import Optimizer
+
+# https://github.com/mlcommons/training/blob/master/image_classification/tensorflow2/lars_optimizer.py
+class LARS(Optimizer):
+  def __init__(self, params: List[Tensor], lr, momentum=0.9, weight_decay=1e-4, eta=0.001, eps=0.0, skip_list=None, nesterov=False, track_gnorm=False):
```

Comment:
what's track_gnorm? did not see it in tf version

Reply:
oh, it was a debug flag i forgot to remove, oops